agreements/employment.md

```diff
@@ -0,0 +1,167 @@
+
+8. ### Notice period
+    1. In cases of gross misconduct, you may be dismissed without notice. The following is an indicative, non-exhaustive list of things that might constitute gross misconduct:
+        1. insubordination;
+        2. committing a tort or being convicted for a criminal offence that impacts on your employment;
+        3. fighting or assaulting another person or causing another person to fear violence;
+        3. deliberately damaging property;
+        4. dishonesty, theft or fraud;
+        5. deliberately or recklessly disclosing confidential information or personal data to a third party;
+        6. willfully or seriously breaching the terms of this contract.
+    2. Other than in cases of gross misconduct, your notice period:
+        1. during your probationary period is two weeks;
+        2. following your probationary period up to the second anniversary of your commencement date is six weeks;
+        3. after the second anniversary of your commencement date, six weeks plus an additional week for each year of service up to a maximum of 12 weeks.
+    3. These periods apply to both you and the company, irrespective of who is giving notice to whom.
+    4. Notice to quit will only be effective if given in writing, which includes by e-mail.
+    5. Once notice has been given (whether from you to us, or from us to you), we reserve the right, at any time after the notice was given while you are still employed with us, to give you pay in lieu of notice. This means that we will pay you, as a lump sum, the salary that would otherwise be payable to you during the unexpired portion of your notice period.
+
+9. ### Holiday entitlement
+    1. We operate a holiday year which runs from 1st July to 30th June in the following year.
+    2. Leave accrues throughout the year according to the formula given below, resulting in 33 days paid leave for full time staff and a pro rata proportion for those working part time.
+    3. Unless notified otherwise you will be required to take all Bank Holidays and Public Holidays as leave and those days are counted in the total of 33 days.
+    4. You may carry over a maximum of five days' leave from one holiday year to the next.
+    5. At any time in a leave year, your accrued leave entitlement may be calculated as follows: 2.08 x Number of days worked in the year to date / 21.75
+    6. The amount of leave you may take will be the total leave accrued, rounded up to the nearest half a day, minus any leave you have already taken in the leave year.
+    7. We will usually permit you to take up to five days leave in advance of it being accrued.
+    8. If you wish to take leave, you must obtain approval (which will not be unreasonably withheld) and give notice of:
+        1. 28 days for a period of leave longer than one day;
+        2. seven days for a period of one day or less.
```

Comment:
I agree with Duncan (although I'm usually able to give lots of notice for holiday so this is probably okay for me). Could it be up to 3 days holiday is okay to give less notice? It would be useful to have more than a week's notice for someone taking a week off though...